py3status/formatter.py

```diff
@@ -418,8 +418,8 @@ def set_param(param, value, key, block, format=''):
             if not item.get('full_text') and not item.get('separator'):
                 continue
             # colors
-            color_this = str(item.get('color'))
-            if color_this and color_this[0] != '#':
+            color_this = item.get('color')
+            if color_this and str(color_this[0]) != '#':
```

Comment:
Atm  i can't reproduce this. But next time when it happens, it now gives more info.